src/formats/jsonLd.ts

```diff
@@ -19,8 +19,14 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-import { JsonLdParser } from "jsonld-streaming-parser";
-import { IriString } from "../interfaces";
+import * as jsonld from "jsonld";
```

Comment:
Is there a reason to prefer this syntax over explicitly importing functions from the `jsonld` package ? I have no issue with the current line, I'm just curious :)

Reply:
I was playing around when looking into some mocking that wasn't working. But, switched back now to the specific named import and all ok, so have reverted to that